### PR TITLE
Retry reconciletag

### LINKED ISSUE #89: Add retries to retag script

### PR DESCRIPTION
**Pull request by cftorres**
@nkubala 
fixes #89

### CODE REVIEW

**Comment by cftorres** [comment]
I signed it!

**Comment by sharifelgamal** [comment]
There are some formatting issues that are causing those travis tests to fail. Take a look at https://travis-ci.org/GoogleCloudPlatform/runtimes-common/builds/240950357 to see what they are.

File: reconciletags/reconciletags.py [hunk 1]
```diff
@@ -25,8 +25,17 @@
...
+
 
 class TagReconciler:
+    def retry_on_calledprocesserror(exception):
+        return isinstance(exception, subprocess.CalledProcessError)
+
+    def retry_on_jsonerror(exception):
+        return isinstance(exception, ValueError)
+
+    @retry(retry_on_exception=retry_on_calledprocesserror)
```
Comment (nkubala) [inline reconciletags/reconciletags.py]: let's put a backoff on these retries, and only try a few times before we just give up completely. maybe wait 2-4 seconds between each try, and try something like 4-5 times?

**Comment by nkubala** [comment]
Travis is failing because it needs the `retrying` package installed. Can you add that to the `requirements.txt` file in the root directory?

**Comment by cftorres** [comment]
I signed it

**Comment by cftorres** [comment]
I signed it

**Comment by cftorres** [comment]
I signed it

**Comment by aaron-prindle** [comment]
The CLA issue might be caused with the email you are using or your email not being associated with all of your git commits.  Make sure you registered your google email with Github (not a personal email), you have your email setup on your local git: https://help.github.com/articles/setting-your-email-in-git/ and that all of your commits have your email associated with them.  You may need to squash your commits into a new commit in order to do this.

**Comment by aaron-prindle** [comment]
is this PR still pending?  should we close this?

**Pull request closed without merging**